doc/bt/BC_COMMANDER_DESTROY.bt

```diff
@@ -2,12 +2,12 @@
 //--- 010 Editor v6.0.2 Binary Template
 //
 // File: BC_COMMANDER_DESTROY.bt
-// Author: exec
-// Revision:
+// Author: exec, celophi
+// Revision: i170175
 // Purpose:
 //--------------------------------------
 
 short op;
 int sequence;
 
-byte index;
+__int64 characterId;
```

Comment:
Why do you prefix the int64s with underscores? Not that that doesn't work, but it's not necessary.

Reply:
Oh. I was under the impression that it was. I can fix those.

Reply:
You can if you want, otherwise I will just run a replace all over the files some time.